lib/bolt_server/config.rb

```diff
@@ -53,8 +56,18 @@ def load_config(path)
       raise "Could not find bolt-server config at #{path}" if parsed_hocon.nil?
 
       parsed_hocon = parsed_hocon.select { |key, _| CONFIG_KEYS.include?(key) }
-      @data = @data.merge(parsed_hocon)
 
+      ENV_KEYS.each do |key|
```

Comment:
I think this should probably be a different method from loading config from a file.